File: Matury CKE/2021 Czerwiec/Programowanie/C++/zad4_1.cpp
#include <iostream>
#include <fstream>
#include <ctype.h>

using namespace std;

int main()
{
    ifstream dane("C:/Matury-Informatyka/Matury CKE/2021 Czerwiec/DANE/napisy.txt");
    // ifstream dane("C:/Matury-Informatyka/Matury CKE/2021 Czerwiec/DANE/przyklad.txt");
    int charsDigitCount = 0;

    string text;
    while(dane >> text){
        for(auto a: text){
            if(isdigit(a)){
                charsDigitCount += 1;
            }
        }
    }

    cout << "4.1. " << charsDigitCount;
    return 0;
}

File: Matury CKE/2021 Czerwiec/Programowanie/Python/zad4_1.py
# dane=[i.strip() for i in open("../../DANE/przyklad.txt")]
dane=[i.strip() for i in open("../../DANE/napisy.txt")]

licznik=0
for napis in dane:
    for znak in napis:
        if znak.isdigit():
            licznik+=1

print(licznik)